lib/sign.js

```diff
@@ -101,7 +106,16 @@ function sign (program, options, config) {
       return result;
     });
 
-  }).then(function(result) {
+  });
+}
+
+
+function signCmd (program, options, config) {
+  return when.promise(function(resolve, reject) {
+    resolve(cmd.validateProgram(program));
+  }).then(function () {
+    return sign(_.assign(options, program), config);
```

Comment:
for safety, this should be `_.assign({}, options, program)` so that it doesn't modify the input object.